cmd/microcloud/main_init_preseed.go

```diff
@@ -519,9 +519,9 @@ func (p *Preseed) Parse(s *service.Handler, c *initConfig) (map[string]InitSyste
 	// If an uplink interface was explicitly chosen, we will try to set up an OVN network.
 	explicitOVN := len(ifaceByPeer) > 0
 
-	cephInterfaces := map[string]map[string]service.DedicatedInterface{}
+	addressedInterfaces := map[string]map[string]service.DedicatedInterface{}
```

Comment:
If the same interface is selected for both the OVN underlay and Ceph would it be helpful to print a message saying the same interface was selected twice?

Not that this isn't a valid setup, but if you proactively select dedicated interfaces, you might not want to mix those two.

Reply:
Good idea, actually I first tried to subtract the selected interface from availability if it was chosen for the other service, but it's technically a valid setup (since by default everything uses the microcluster network anyway). 

I'll add a warning here.